app/js/controllers/rules_controller.js

```diff
@@ -6,20 +6,27 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
 
   $scope.rule_id = parseInt($routeParams.id, 10);
   $scope.pr_ch_options = ["All rules"];
+
+  $scope.currentPage = 1;
+  $scope.pageSize = 10;
+  $scope.maxSize = 5;
+
   if ($scope.rule_id) {
-    // history of a specific rule
-    Rules.getHistory($scope.rule_id)
-    .success(function(response) {
-      // it's the same rule, but this works
-      $scope.rules = response.rules;
-    })
-    .error(function() {
-      console.error(arguments);
-      $scope.failed = true;
-    })
-    .finally(function() {
-      $scope.loading = false;
-    });
+    console.log("yes");
+    // // history of a specific rule
+    // Rules.getHistory($scope.rule_id, $scope.pageSize, $scope.currentPage)
+    // .success(function(response) {
+    //   // it's the same rule, but this works
+    //   $scope.rules = response.rules;
+    //   $scope.rules_count = response.count;
+    // })
+    // .error(function() {
+    //   console.error(arguments);
+    //   $scope.failed = true;
+    // })
+    // .finally(function() {
+    //   $scope.loading = false;
+    // });
   } else {
```

Comment:
I commented this because it gets run after the 'watch' bit at line 63 so it is i think redundant